canonical-web-and-design/maas-ui
# refactor(ui): make filter accordion reusable

## Done

- Update the filter accordion to be a reusable component.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Visit the machine list.
- Check that you can use the filter accordion to change the filters.

## Fixes

Fixes: canonical-web-and-design/app-squad#134.
